third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -308,6 +308,10 @@ subnetwork in which the cluster's instances are launched.
 * `enable_intranode_visibility` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
     Whether Intra-node visibility is enabled for this cluster. This makes same node pod to pod traffic visible for VPC network.
 
+* `cluster_telemetry` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
```

Comment:
Since this is a block, can you add a "structure is documented below" and then "the `cluster_telemetry` block supports" later on? you can see other places in this doc or in any of our generated docs where we do similar constructs.

Reply:
done